src/styles/isomer-cms/elements/modal.scss

```diff
@@ -58,3 +60,50 @@
     }
   }
 }
+
+.modal-warning {
+  width: 50%;
+  background: white;
+  padding: 30px;
+  box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.2);
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  overflow-y: scroll;
+  position: absolute;
+
+
+  .modalHeader{
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+
+    h1{
+      font-size: 24px;
+    }
+
+    .bx{
+      font-size: 30px;
+    }
+  }
+
+  .modalContent {
+    padding: 20px 0px 10px 0px;
+    height: auto;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    flex: 1;
+  
+    .modalButtons {
+      padding-top: 20px;
+      display: flex;
+      flex-direction: row;
+      justify-content: flex-start;
+    }
+  }
```

Comment:
`.modalContent` and `.modalButtons` are identical to the CSS classes defined higher up in the `modal.scss` file. Instead of the duplication of code, you should abstract these subclasses and use them in both `.modalWarning` and `.modal`

Reply:
Actually, is the only diff between a normal modal and the warning modal the difference in height? If so, you can create a new CSS selector for `modalWarning`, where the height is set differently.

Reply:
yeah you're right i'll just flatten out the classes